test/env.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  notifyApiKey: 'smoke_test_api_key-822632b6-1b07-44ef-9fe0-24dc9bb99272-45cce8e4-7daa-4233-89f3-7e53f7bde3e8',
+  notifyEmailTemplateId: 'ba96327f-d372-4589-90f5-e280a37c85fc',
+  // https://docs.notifications.service.gov.uk/node.html#smoke-testing
+  notifyEmailAddress: 'simulate-delivered@notifications.service.gov.uk'
```

Comment:
Smoke test email address given by GOV.UK Notify -- should the smoke API key (which is owned by us) be removed?